syntax="proto3";

package bidirectional_databroker;

message BrokerRequest{
    int32 id = 1;
    float sensor1 = 2 ;
    float sensor2 = 3 ;
    float sensor3 = 4 ;
    float sensor4 = 5 ;
}

message BrokerResponse{
    int32 id = 1;
    bool prediction = 2;
}

service BrokerService{
    rpc BidirectionalStreaming(stream BrokerRequest) returns (stream BrokerResponse);
}